<ui version="4.0" >
 <class>dlgDataDialog</class>
 <widget class="QDialog" name="dlgDataDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>313</width>
    <height>634</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Data Summary</string>
  </property>
  <widget class="QTableWidget" name="tblAttributes" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>420</y>
     <width>311</width>
     <height>211</height>
    </rect>
   </property>
   <property name="selectionMode" >
    <enum>QAbstractItemView::NoSelection</enum>
   </property>
   <column>
    <property name="text" >
     <string>Cell Type</string>
    </property>
   </column>
   <column>
    <property name="text" >
     <string>Attribute</string>
    </property>
   </column>
   <column>
    <property name="text" >
     <string>Value</string>
    </property>
   </column>
  </widget>
  <widget class="QTableWidget" name="tblCellType" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>210</y>
     <width>311</width>
     <height>201</height>
    </rect>
   </property>
   <property name="selectionMode" >
    <enum>QAbstractItemView::NoSelection</enum>
   </property>
   <column>
    <property name="text" >
     <string>Cell Type</string>
    </property>
   </column>
   <column>
    <property name="text" >
     <string>Population</string>
    </property>
   </column>
  </widget>
  <widget class="QTableWidget" name="tblMolType" >
   <property name="enabled" >
    <bool>true</bool>
   </property>
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>311</width>
     <height>201</height>
    </rect>
   </property>
   <property name="selectionMode" >
    <enum>QAbstractItemView::NoSelection</enum>
   </property>
   <column>
    <property name="text" >
     <string>Mol. Type</string>
    </property>
   </column>
   <column>
    <property name="text" >
     <string>Moles/ml</string>
    </property>
   </column>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
